package.json

```diff
@@ -47,15 +47,16 @@
     "event-stream": "3.3.4",
     "express": "4.14.0",
     "fs-extra": "1.0.0",
+    "fuzzy": "0.1.3",
     "irc-framework": "2.5.3",
+    "ldapjs": "1.0.1",
```

Comment:
I keep seeing a lot of PRs affecting this package due to bad sorting, I'll make a separate PR to fix ite.